src/actions/auto-assign.ts

```diff
@@ -80,54 +80,36 @@ export async function run(): Promise<void> {
       )}`,
     );
 
+    const absentEmployeesToday: string[] = inputs.checkReviewerOnSage
+      ? await getEmployeesWhoAreOutToday({
+          sageBaseUrl: inputs.sageUrl,
+          sageToken: inputs.sageToken,
+        })
+      : [];
+
+    const sageUsers = config.sageUsers || {};
+    const absentReviewersLogins = requestedReviewerLogins.filter((reviewer) => {
+      const sageUser = sageUsers[reviewer];
+      return sageUser && absentEmployeesToday.includes(sageUser[0].email);
+    });
```

Comment:
let's move this to separate function away from main code. convertSageEmailsToGithubUsers.
also cover it with unit tests

also wrong logic behind this
***

## pseudo code
```
const employeesOutEmails = await sage...;
const absentReviewersLogins = convertSageEmailsToUsernames({config.dsfasdafs, employeesOutEmails})
```